### ISSUE TITLE
Error extracting i18n tags with Angular 15+

### ISSUE THREAD

**Issue by jaudette790** [mattlewis92/angular-calendar#1671]
**Describe the bug**
When running `ng extract-i18` in project, i get the error:
```
/node_modules/angular-calendar/fesm2020/angular-calendar.mjs - Error: Module build failed (from ./node_modules/@angular-devkit/build-angular/src/babel/webpack-loader.js):
Error: (...)/node_modules/angular-calendar/fesm2020/angular-calendar.mjs: 'column' must be greater than or equal to 0 (columns start

"extract-i18n" failed: undefined

```

**Minimal reproduction of the problem with instructions**
Since this is not a browser issue, did not create a stackblitz.
See [this stackoverflow question with the same issue](https://stackoverflow.com/questions/76362937/getting-column-must-be-greater-than-or-equal-to-0-error-while-trying-to-extrac)

**Screenshots**
No screenshots

**Versions**

- `@angular/core`: 16.0.3
- `angular-calendar`: 0.31.0 (also tried 0.30.1)
- Browser name and version: Node version 16, 18, 20 are all doing the same thing.

**Comment by cmayankc** [comment]
+1

**Comment by geraldspreer** [comment]
+1

**Comment by geraldspreer** [comment]
The error is thrown by babel.js https://unpkg.com/@babel/standalone@7.21.2/babel.js.map 

`const COL_GTR_EQ_ZERO = '`column` must be greater than or equal to 0 (columns start at column 0)';`

**Comment by DerZyklop** [comment]
Have the same on Angular 15. @jaudette790 you could change 'Angular 16' to 'Angular 15+' in the title.

**Comment by DerZyklop** [comment]
And i confirm @geraldspreer. Error is 
```shell
Error: /Users/nils/S/plano/scheduler.frontend.ng2/node_modules/angular-calendar/fesm2020/angular-calendar.mjs: `column` must be greater than or equal to 0
```

**Comment by jaudette790** [comment]
> Have the same on Angular 15. @jaudette790 you could change 'Angular 16' to 'Angular 15+' in the title.

I updated the title, but I did not get the error until I updated to Angular 16

**Comment by mattlewis92** [comment]
Hey folks, I just took a quick look at this issue and this seems like a bug with angular rather than anything I'm able to fix my end. You can see there are other people running into the problem and that aren't using this package.

For example:
* https://github.com/angular/angular-cli/issues/25403
* https://github.com/angular/angular/issues/50777

**Comment by mattlewis92** [comment]
One temporary workaround that I found is to delete the sourcemap before extracting i18n:

```
rm node_modules/angular-calendar/fesm2020/angular-calendar.mjs.map
```

**Comment by geraldspreer** [comment]
Thank you @mattlewis92 for providing this info to temporary fix this problem.

**Comment by mattlewis92** [comment]
I've filed an issue on the angular CLI repo with steps to reproduce, hopefully they can find a proper fix soon 🙏 

https://github.com/angular/angular-cli/issues/25413

**Comment by kukjevov** [comment]
It is a problem of Babel which is already under resolution, but not closed yet https://github.com/babel/babel/issues/15712

**Comment by kukjevov** [comment]
@mattlewis92 can you verify whether `babel` version `7.22.7` solves issue also for you?

https://github.com/angular/angular-cli/issues/25414#issuecomment-1623448186

**Comment by mattlewis92** [comment]
[Verified](https://github.com/mattlewis92/angular-calendar-ng16-test/commit/1168a6de51413a0b65da62ddf461800876e4d1d5) that the issue is fixed in angular CLI `16.1.4`!
